portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomLanguage() {
   const languageContainer = document.getElementById('language-container');
   languageContainer.innerText = language;
 }
+
+async function showContent() {
```

Comment:
Can you add a function-level comment for this?

Reply:
Done!